### DISCUSSION TITLE
Loading problem after update to 10.x

### DISCUSSION THREAD

**Discussion by Hbitvof** [pimcore/pimcore#12122] [Q&A / Support]
Hi,

I have a strange issue that occured after updating from 6.9 to 10.x. Every day when first starting Pimcore the site is not loading, after refreshing Pimcore is loading.

The first time I get these errors:

![image](https://user-images.githubusercontent.com/41229269/168468071-bed5b269-95c8-4d88-abf5-5907dbd05373.png)

There is also a warning about an expired cookie:

Cookie “_pc_tvs” has been rejected because it is already expired.

I already cleared all browser cache, but this is not solving the problem. Is this a known issue and what can I do to solve this?

**Comment by Hbitvof** [comment]
Anyone?

**Reply by kingjia90** [reply]
I am having similar problems, converting this in a issue, feel free to add any useful details here #12165

**Comment by Hbitvof** [comment]
Any update on this problem? It's getting pretty annoying to refresh every time starting PIM.
